README.md

```diff
@@ -11,9 +53,9 @@ We will use the system `ls` command for running, and we will assume it is locate
 ```console
 git clone https://github.com/unikraft/app-elfloader/
 cd app-elfloader/
```

Comment:
```suggestion
git clone https://github.com/unikraft/app-elfloader/ elfloader
cd elfloader/
```

The `APP_NAME` is not part of the defconfigs, so we need to have the directory named `elfloader` (or add the `CONFIG_UK_NAME="elfloader"` to the defconfigs)